etc/base.yaml

```diff
@@ -502,8 +417,30 @@ radar_push_endpoint:
   adminProperties: {}
   garmin:
     enabled: true
-    consumerKey: ""
-    consumerSecret: ""
+
+# --------------------------------------------------------- 40-realtime-analyses.yaml ---------------------------------------------------------
+
+radar_jdbc_connector_agg:
+  _install: false
+  _chart_version: 0.4.0
+  replicaCount: 1
+  timescaledb:
+    host: timescaledb-postgresql-hl
```

Comment:
should this be updated in line with the other jdbc connector (so key `jdbc` instead of `timescaledb` and the `url` of connection)